test/snippet/alignment/configuration/align_cfg_gap_cost_affine_example.cpp

```diff
@@ -0,0 +1,12 @@
+#include <seqan3/alignment/configuration/align_config_gap_cost_affine.hpp>
+
+int main()
+{
+    // Configuration with linear gaps.
```

Comment:
This might be a bit more exact :)
```suggestion
    // Configuration with linear gap costs.
```